lib/system.g

```diff
@@ -564,6 +564,33 @@ BIND_GLOBAL("ARCH_IS_UNIX",function()
   return not ARCH_IS_WINDOWS();
 end);
 
+#############################################################################
+##
+#F  ARCH_IS_WSL()
+##
+##  <#GAPDoc Label="ARCH_IS_WSL">
+##  <ManSection>
+##  <Func Name="ARCH_IS_WSL" Arg=''/>
+##
+##  <Description>
+##  tests whether &GAP; is running on a Windows system inside the
+##  'Windows Subsystem for Linux'. Note that in this case
+##  <Ref Func="ARCH_IS_UNIX"/> will be <C>true</C>, and in most situations
```

Comment:
```suggestion
##  <Ref Func="ARCH_IS_UNIX"/> will be <K>true</K>, and in most situations
```

Reply:
I'm happy to fix it, although I coped what I wrote from about 20 lines higher up (I'll fix that too)